index.html

```diff
@@ -7,7 +7,7 @@
     <link rel="stylesheet" href="styles/style.css">
```

Comment:
I'm not sure why, but I'm not seeing any fonts that aren't browser-standard. I think you also need this in your `head` section: 
```<link href="https://fonts.googleapis.com/css?family=Galada|Heebo|Libre+Baskerville" rel="stylesheet">```